pwndbg/aglib/elf.py

```diff
@@ -255,10 +253,9 @@ def entry() -> int:
     """
     Return the address of the entry point for the main executable.
     """
-    if pwndbg.dbg.is_gdblib_available():
-        entry = pwndbg.auxv.get().AT_ENTRY
-        if entry:
-            return entry
+    entry = pwndbg.auxv.get().AT_ENTRY
+    if entry:
```

Comment:
Nitpick, but this should probably be `if entry is not None:` because if `AT_ENTRY` were to be 0, we won't return it properly? :D 

This is 100% edge case and very unlikely to ever happen... One would need a linux with `vm.mmap_min_addr=0` sysctl setting as well as a really custom binary with entrypoint set to address 0. Honestly, I don't think this is ever possible, so maybe let's leave this as it is ;).